docs/backup-restore.md

```diff
@@ -1,32 +1,104 @@
+Scalar DB Schema Loader enables PITR and auto-scaling by default.
+
+## Restore Backup
+
+### JDBC databases
+
+You can restore the backup with your favorite way for JDBC databases.
+You can use `mysql` command in MySQL and `psql` command in PostgreSQL to achieve that.
+If you use Amazon RDS (Relational Database Service) or Azure Database for MySQL/PostgreSQL,
+you can restore to any point within the backup retention period with the automated backup feature.
+
+### Cassandra
+
+To restore the backup, you should stop the Cassandra and clean the `data`, `commitlogs`, and `hints` directories then restore the backup copies of all the nodes of the cluster, and start the cluster.
+
+To avoid mistakes when doing restore operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
+You can restore to any point using `CLUSTER-ID` and `SNAPSHOT-ID`.
+Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
+
+### Cosmos DB
+
+To restore a backup, please follow the [azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/restore-account-continuous-backup#restore-account-portal) and change the default consistency to `STRONG` after restoring the backup.
+It is recommended to use the midtime of paused duration as a restore point as we explained earlier.
+
+### DynamoDB
+
+You can restore the table to a point in time using the DynamoDB console or the AWS Command Line Interface (AWS CLI). The point-in-time recovery process restores to a new table.
+The tables can only be restored one by one.
+
+You can restore tables from the [Amazon DynamoDB console](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/PointInTimeRecovery.Tutorial.html) using the following steps.
+
+* Select the midtime of paused duration as the restore point.
+* Restore with PITR of table A to another table B
+* Take a backup of the restored table B (assume the backup is named backup B)
+* Remove table A
+* Create a table named A with backup B
+
+Note that you need to follow the above steps because it assumes the application requires tables with the same names as before. In DynamoDB, a table can only be restored with an alias, so you can restore the table with an alias and delete the original table and rename the alias to the original name.
+
+If you want to restore multiple tables with a single command, you can create a script to restore multiple tables using the AWS CLI commands.
 
-## Scalar DB server
+DynamoDB will not enable features like PITR and auto-scaling by default after restoring. It is recommended to enable point-in-time recovery (PITR) for continuous backup creation.
+Auto-scaling dynamically adjusts provisioned throughput capacity on your behalf in response to actual traffic patterns. You can enable this feature if you need it.
```

Comment:
other configurations are all reset as well?
Then we should say in a different way

Reply:
From [AWS documentation](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/CreateBackup.html#CreateBackup_HowItWorks-restore) and [blog](https://aws.amazon.com/blogs/aws/new-amazon-dynamodb-continuous-backups-and-point-in-time-recovery-pitr/) post, configurations like Auto scaling policies, AWS Identity and Access Management (IAM) policies, Amazon CloudWatch metrics and alarms, Tags, Stream settings, Time to Live (TTL) settings, Point-in-time recovery settings must be set manually after restoring the tables.
I have updated in different way